File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/FullScreenScannerFragmentActivity.java
package me.dm7.barcodescanner.zxing.sample;


import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import me.dm7.barcodescanner.zxing.sample.databinding.ActivityFullScreenScannerFragmentBinding;

public class FullScreenScannerFragmentActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle state) {
        super.onCreate(state);

        ActivityFullScreenScannerFragmentBinding binding = ActivityFullScreenScannerFragmentBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/fullscanner/FullScreenScannerFragmentActivity.kt
package me.dm7.barcodescanner.zxing.sample.fullscanner

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import me.dm7.barcodescanner.zxing.sample.databinding.ActivityFullScreenScannerFragmentBinding

class FullScreenScannerFragmentActivity : AppCompatActivity() {

    override fun onCreate(state: Bundle?) {
        super.onCreate(state)

        val binding = ActivityFullScreenScannerFragmentBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}
